pkg/util/hostname/providers.go

```diff
@@ -12,13 +12,14 @@ import (
 	"expvar"
 	"fmt"
 
+	"github.com/DataDog/datadog-agent/comp/core/hostname/hostnameinterface"
 	"github.com/DataDog/datadog-agent/pkg/util/cache"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 const (
-	configProvider  = "configuration"
-	fargateProvider = "fargate"
+	configProvider  = hostnameinterface.ConfigProvider
```

Comment:
What is the reason to introduce this alias as `configProvider` is  used at 2 locations?